### PR TITLE
remove redundant code

### PR DESCRIPTION
**Pull request by wankai**
scoped_malloc_t ser_buf_ already support move semantic

### CODE REVIEW

**Comment by danielmewes** [comment]
Good catch. Thank you for the pull request @wankai .

Would you mind signing our CLA if you haven't done so yet? http://rethinkdb.com/community/cla/
Please let me know once you have signed it.

**Comment by wankai** [comment]
@danielmewes  CLA done.

File: src/main.cc [hunk 1]
```diff
@@ -15,8 +15,6 @@ int main(int argc, char *argv[]) {
     setrlimit(RLIMIT_CORE, &core_limit);
 #endif
 
-    startup_shutdown_t startup_shutdown;
```
Comment (danielmewes) [inline src/main.cc]: This isn't redundant, since it has side effects.

**Comment by danielmewes** [comment]
Thank you.
The ser_buffer change looks good, but `startup_shutdown_t` is required.

**Comment by wankai** [comment]
sorry, I didn't notice it's a RAII

**Comment by danielmewes** [comment]
Oh the first part was good. Feel free to open a new pull request or re-open this with the `startup_shutdown_t` commit reverted.

**Pull request closed without merging**